src/utils/pointerEvents.js

```diff
@@ -67,60 +66,56 @@ function isSimplaName(name) {
 }
 
 /**
- * Check if the given element is a Simpla Element
- * @param  {Node}  node   Node to check
- * @return {Boolean}      True if node is a Simpla Element
+ * Returns an array of elements in a nodeList
+ * @param  {NodeList} list Nodelist to filter
+ * @return {HTMLElement[]} Array of element nodes
  */
-function isSimplaElement(node) {
-  if (isSimplaName(node.localName)) {
-    return true;
-  }
-
-  return node.getAttribute && isSimplaName(node.getAttribute('is'));
+function elementsIn(list) {
+  return Array.prototype.filter.call(list, node => node.nodeType === 1);
 }
 
 /**
- * Checks if the given node is an editable element. Editable means the element
- *  is a Simpla Element and it is not a special ignored element e.g. simpla-admin
- * @param  {Node}   node  Node to check if it's an editable element
- * @return {Boolean}      True if editable element, false otherwise
+ * Check if the given element is a Simpla Element
+ * @param  {HTMLElement}  element  Element to check
+ * @return {Boolean}      True if node is a Simpla Element
  */
-function isEditableElement(node) {
-  const ignored = (name) => ADMIN_ELEMENTS.includes(name),
-        isEditableName = (name) => isSimplaName(name) && !ignored(name);
-
-  if (isEditableName(node.localName)) {
+function isSimplaElement(element) {
+  if (isSimplaName(element.localName)) {
     return true;
   }
 
-  return isEditableName(node.getAttribute && node.getAttribute('is'));
+  return isSimplaName(element.getAttribute('is'));
 }
 
 /**
- * Get all editable elements in the given list.
- * @param  {(NodeList|Node[])} list List of nodes to check
- * @return {HTMLElement[]}          Array of editable elements
+ * Get all editable elements in the given list
+ * @param  {Array} elements Array of elements to check
+ * @return {HTMLElement[]}  Array of editable elements
  */
-function editableElementsIn(list) {
-  return Array.prototype.filter.call(list, isEditableElement);
+function ignoredElementsIn(elements) {
+  const shouldIgnore = element => isSimplaElement(element) ||
```

Comment:
Yeah fair on premature optimisation, if it becomes sluggish, then we can look at them.